src/content/structured/get-started/design-principles.mdx

```diff
@@ -48,4 +48,4 @@ Enable feedback on processes and content from the entire community. Allow contri
 
 ## Responsible
 
-We factor environmental sustainability into our decisions - minimising our resource usage and enabling the community to do the same.
+We factor environmental sustainability into our decisions—minimising our resource usage and enabling the community to do the same.
```

Comment:
Another case where spaces around the dash feel natural - this ends up looking like a hyphenated phrase rather than two clauses of a sentence.  You could replace it with a comma?

Reply:
Happy to replace with a comma instead of an em rule.